app/src/main/res/values/localized.xml

```diff
@@ -480,4 +480,12 @@ After pressing on the button, switch the app filter from \"Not optimized\" to \"
     <string name="servant_enhancement_none_selected">No Servant Selected. Please select a Servant before proceeding.</string>
     <string name="servant_enhancement_aborted">Enhancement was aborted</string>
     <string name="note">"Note:"</string>
+
+    <string name="p_battle_config_raid">"Raid"</string>
```

Comment:
```suggestion
```

Reply:
"Raid Delay" is too long for the UI.

I'm thinking of adding the animation skip settings to the Raid button so they're combined.

Or create a new Raid section in the battle config where you can configure the values directly instead of in a popup.